backend/src/main/java/ai/verta/modeldb/name_generator/NamesGenerator.java

```diff
@@ -0,0 +1,1179 @@
+package ai.verta.modeldb.name_generator;
```

Comment:
Did you copy this from somewhere?

Reply:
Yes. It's copy from the Docker open source project. see https://github.com/VertaAI/modeldb/pull/1657/files#diff-13ba750a8f7a820dab9e8cc12d78b70526a37fa4f3513014d5eb54f3f8c96181R5-R8

Reply:
That link doesn't point to the original.

We cannot copy code from open source willy nilly, unfortunately. Can we use it as a library? If not, we have to copy into a separate folder (using their own package structure), satisfy their license requirements and give proper attribution. Otherwise we're liable.